packages/analytics-js-common/src/types/ErrorHandler.ts

```diff
@@ -16,4 +16,5 @@ export interface IErrorHandler {
   ): void;
   leaveBreadcrumb(breadcrumb: string): void;
   notifyError(error: Error): void;
+  attachErrorListeners(): void;
```

Comment:
need to fix the type of onError now that a new argument "errorType" is added

Reply:
Addressed